marimbabot_vision/scripts/generate_data.py

```diff
@@ -1,9 +1,10 @@
+import argparse
```

Comment:
Maybe add a shebang.

Reply:
then I would need to add it to each python file

Reply:
Yeah. It is just a convenience thing, like the argparse interface, so we can call the script without python3 xxxx.py